### DISCUSSION TITLE
Copilot suggestion not change all selection words

### DISCUSSION THREAD

**Discussion by oliverdettenborn** [community/community#6913] [Copilot Conversations] [answered]
When I selection more then one word and accept a copilot suggestion, only the current word is change, instead of all selected words.

To reproduce:
1 - Select all occurrence for a word with control + F2
2 - Accepted a suggestion
3 - Only the first word is changed

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Can you provide us with a video screen capture of reported behaviour? Thanks!

**Reply by oliverdettenborn** [reply]
https://user-images.githubusercontent.com/70967247/139284588-1020f48d-9151-4702-ae7c-49a5fd92589a.mp4
